src/stores/settings.ts

```diff
@@ -32,6 +32,9 @@ export const useSettingsStore = defineStore('uiSettings', {
       useHeikinAshiCandles: false,
       notifications: notificationDefaults,
       profitDistributionBins: 20,
+      colorPreference: 'greenUp',
+      colorProfit: '#12bb7b',
+      colorLoss: '#ef5350',
```

Comment:
i MAY would extract colors from the settings-store into a separate store - but i guess having it here can be fine, too.